pkg/ctrl/example_test.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ctrl_test
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-sigs/kubebuilder/pkg/ctrl"
+	"k8s.io/api/apps/v1"
+)
+
+func ExampleController() {
```

Comment:
You can also add docs to Examples (in case you forgot)

Reply:
Done.